buildconfig/pygame-stubs/transform.pyi

```diff
@@ -1,50 +1,39 @@
-from typing import Tuple, List, Union, Optional, Sequence
-from pygame.surface import Surface
-from pygame.math import Vector2
+from typing import Optional, Sequence, Tuple, Union
+
 from pygame.color import Color
```

Comment:
I guess these multiple import lines could be changed to:
`from pygame import Color, Rect, Surface, Vector2`

Reply:
Changing all the imports to be import the module and not the symbol. So, it will be:
from . import color

and when using it, it will be color.Color - per guidance